ajout du pont entre clustering et allennlp

AnthonyGigerich opened the pull request:
Prend en entrée une liste d'arrété concernant des terrasses sous format json (sortie du clustering/recuperation deterministe) et génère des fichiers .txt sous le format d'entrée d'AllenNLP

AnthonyGigerich committed:
ajout du pont entre clustering et allennlp

Prend en entrée une liste d'arrété concernant des terrasses sous format json et génère des fichiers .txt sous le format d'entrée d'AllenNLP

moreymat commented on src/prepare_token.py:
```diff
@@ -0,0 +1,144 @@
+
+#entrée: un mot avec une apostrophe | ie. "l'article"
+#sortie: deux mots séparés par l'apostrope | ie. "l'" et "article"
+def split_apostrophe(word):
```
C'est un détail mais j'aurais plutôt utilisé https://docs.python.org/3.3/library/stdtypes.html?highlight=split#str.index ou https://docs.python.org/3.3/library/stdtypes.html?highlight=split#str.find ou str.partition, voire str.split .
Mais c'est juste un détail et si ça marche comme ça, on laisse.

AnthonyGigerich replied:
En effet, c'est beaucoup plus propre merci.

moreymat commented on src/prepare_token.py:
```diff
@@ -0,0 +1,144 @@
+def prepare_tokens(arrete):
+
+    f = open(
+        r"data/test/" + arrete["title"][0]["text"][0:17] + ".txt",
```
Pourquoi 17 ?

AnthonyGigerich replied:
les articles qui nous concerne sont tous sous un format "2017_01461_VDM".
C'est assez spécifique je pourrais faire en sorte de le généraliser en arrêtant au premier espace par exemple

moreymat replied:
OK, alors pas besoin de changer le code, il suffit d'ajouter une ligne de commentaire avec cette justification.

moreymat replied:
On généralisera quand on en aura vraiment besoin.

moreymat reviewed (commented):
Mes commentaires ne sont pas bloquants et ne constituent pas une priorité vu le temps qu'il reste.
Vous pouvez fusionner la branche quand vous voulez !

AnthonyGigerich committed:
Update prepare_token.py

Prise en compte des commentaires (str.index et commentaire pour le nom du fichier)

moreymat commented:
OK pour moi, fusionnez quand vous voulez.
